Pod/Core/GDGEntityMap_Protected.h

```diff
@@ -0,0 +1,14 @@
+//
+//  GDGEntityMap_Protected.h
+//  GoldDigger
+//
+//  Created by Felipe Lobo on 4/14/16.
+//
+
+#import "GDGEntityMap.h"
+
+@interface GDGEntityMap ()
+
+@property (strong, nonatomic) NSMapTable *reverseFromToTable;
```

Comment:
This property is also declared as private on `GDGEntityMap` and this header file is never used.